content/es/partners/_index.html

```diff
@@ -0,0 +1,91 @@
+---
+title: Partners
+bigheader: Kubernetes Partners
+abstract: Expandiendo el ecosistema de Kubernetes.
+class: gridPage
+cid: partners
+---
+
+<section id="users">
+    <main>
+        <h5>Kubernetes trabaja con socios para crear una base de código sólida y vibrante que admita un espectro de plataformas complementarias.</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>Kubernetes Certified Service Providers</b>
+              </h5>
+              <br>Proveedores de servicios con amplia experiencia ayudando a las empresas a adoptar con éxito Kubernetes.
```

Comment:
"adoptar Kubernetes con éxito" suena mejor?

Reply:
Fixed in https://github.com/kubernetes/website/pull/13543/commits/5f332b24a49d9abf288b2f2d661fb55e5485da55